_posts/2017-10-09-omero-5-4-0.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Release of OMERO 5.4.0
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.4.0.
+---
+Today we are releasing OMERO 5.4.0, a full production-ready release of OMERO;
```

Comment:
A bit of a pet hate for every one of our release announcements.  Why do we write "we are releasing" rather than the definite "we have released"?  It leaves me uncertain that the release has been made yet during that day.